projects/errors/models/Error.json

```
@@ -9,19 +9,42 @@
     },
     "title": {
       "type": "string",
-      "description": "A short, human-readable summary of the problem type."
+      "description": "A short, human-readable summary of the problem type (for developers)."
     },
     "status": {
       "type": "integer",
       "description": "The HTTP status code."
     },
     "detail": {
       "type": "string",
-      "description": "A human-readable explanation specific to this occurrence of the problem. "
+      "description": "A human-readable explanation specific to this occurrence of the problem (for developers). "
+    },
+    "endUserMessage": {
+      "type": "object",
+      "description": "A human-readable explanation of the problem, specifically for end-users, in one or more languages. Usually set for domain errors, but not for errors caused by a technical issue in the integration (for example invalid JSON syntax in a request body). An `nl` value is always provided, other languages may be provided depending on the API and its intended audience.",
```

I would repeat the phrase about recommendation to show to end-users here as well:
```suggestion
      "description": "A human-readable explanation of the problem, specifically for end-users, in one or more languages. Typically available for domain errors, but not for errors caused by a technical issue in the integration (for example invalid JSON syntax in a request body). An `nl` value is always provided, other languages may be provided depending on the API and its intended audience. When this property is included, it is strongly encouraged to show this to the end-user.",
```